src/Components/Quizz/PropositionsQuizz.js

```diff
@@ -0,0 +1,96 @@
+import React , {Component} from 'react';
+import './PropositionsQuizz.css';
+import Modal from './Modal'
+
+
+class PropositionsQuizz extends Component {
+
+
+    constructor() {
+        super();
+
+        this.state = {
+            isShowing: false
+        }
+    }
+
+    openModalHandler = () => {
+        this.setState({
+            isShowing: true
+        });
+    }
+
+    closeModalHandler = () => {
+        this.setState({
+            isShowing: false
+        });
+    }
+
+        state = {
+          resultatQuestion : null,
```

Comment:
`resultatQuestion` seems to be a boolean which means you should write false by default. `resultatQuestion`'s naming does not reflect its boolean nature and is written in french instead of english, please fix this.